subprocess/build.gradle

```diff
@@ -7,10 +7,11 @@ import java.time.Instant
 
 plugins {
     id 'java'
-    id "com.github.johnrengelman.shadow" version "8.1.1"
-    id "com.palantir.git-version" version "3.0.0"
+    id 'base'
+    id "com.gradleup.shadow" version "8.3.6"
```

Comment:
Project was moved to the `GradleUp` organization
https://github.com/GradleUp/shadow#gradle-shadow